v1/plugins/logs/encoder_test.go

```diff
@@ -145,12 +152,12 @@ func TestChunkEncoderAdaptive(t *testing.T) {
 		t.Fatalf("Expected %v events but got %v", numEvents, numEventsActual)
 	}
 
-	actualScaleUpEvents := enc.metrics.Counter(encSoftLimitScaleUpCounterName).Value().(uint64)
-	actualScaleDownEvents := enc.metrics.Counter(encSoftLimitScaleDownCounterName).Value().(uint64)
-	actualEquiEvents := enc.metrics.Counter(encSoftLimitStableCounterName).Value().(uint64)
+	actualScaleUpEvents := enc.metrics.Counter(encUncompressedLimitScaleUpCounterName).Value().(uint64)
+	actualScaleDownEvents := enc.metrics.Counter(encUncompressedLimitScaleDownCounterName).Value().(uint64)
+	actualEquiEvents := enc.metrics.Counter(encUncompressedLimitStableCounterName).Value().(uint64)
 
-	expectedScaleUpEvents := uint64(8)
-	expectedScaleDownEvents := uint64(3)
+	expectedScaleUpEvents := uint64(4)
+	expectedScaleDownEvents := uint64(0)
 	expectedEquiEvents := uint64(0)
```

Comment:
Should we assert the expected number of chunks too? I'm guessing these aren't exactly the sum of these events.

Reply:
Updated the test to add more cases, it also checks the sum of the events and the expected IDs of the events to make sure nothing is lost.